.github/workflows/basic.yml

```diff
@@ -228,6 +228,57 @@ jobs:
         --umf-version ${{env.UMF_VERSION}}
         ${{ matrix.shared_library == 'ON' && '--shared-library' || ''}}
 
+  windows-build_hwloc:
+    name: "Windows + static hwloc"
+    env:
+      VCPKG_PATH: "${{github.workspace}}/build/vcpkg/packages/tbb_x64-windows;${{github.workspace}}/build/vcpkg/packages/jemalloc_x64-windows"
+      BUILD_DIR : "${{github.workspace}}/build/${{matrix.build_type}}"
+      INSTL_DIR : "${{github.workspace}}/../install-dir"
+    strategy:
+      matrix:
+        os: ['windows-2019', 'windows-2022']
+        build_type: [Release]
+        shared_library: ['ON']
+        level_zero_provider: ['OFF']
```

Comment:
I find it easier to read jobs when matrix includes only variables that actually have multiple values instead of having constant variables defined here.